src/components/templates/PartyMap/components/Map/MapRoom.tsx

```diff
@@ -88,11 +89,28 @@ export const MapRoom: React.FC<MapRoomProps> = ({
     selectRoom();
   }, [play, selectRoom]);
 
+
+ const handleKeyPress = (e:React.KeyboardEvent<HTMLDivElement>, keyPressFunction: { (): void; (): void; }) => {
+  const enterOrSpace =
+  e.key === "Enter" ||
+  e.key === " " ||
+  e.key === "Spacebar" ||
+  e.which === 13 ||
+  e.which === 32;
+  if (enterOrSpace) {
+    e.preventDefault();
+    keyPressFunction();
+  }
+ }
+
   return (
     <div
+      role="button"
+      tabIndex={0}
       className={containerClasses}
       style={roomInlineStyles}
       onClick={isCovertRoom ? noop : selectRoomWithSound}
+      onKeyPress={isCovertRoom ? noop: (e) => handleKeyPress(e, selectRoomWithSound)}
```

Comment:
We shouldn’t need to explicitly handle key presses here if we use the correct base types that naturally support these navigations. Eg. Button